web/src/AppController.ts

```diff
@@ -32,6 +33,14 @@ class AppController {
     this.loadCount = 0
   }
 
+  componentDidMount() {
```

Comment:
where do you expect these methods to be called? Note that AppController isn't a react component.

Reply:
D'oh...

/me drinks more coffee